### PR TITLE
Feature/logic dev

### PR DESCRIPTION
**Pull request by MarilenaRoque**
# 4th Milestone Requirements Completed :rocket:

### Developed the logic of calculator functionality

Calculate

  - Create a calculate method to process the data
  - Add functionalities for buttons ('AC ', '=', '+/-' and math operators)
  - Use the operating method to make an operation when the sentence is completed.

  Operate

  - Create a method for the basic operations
  - Use Big.js to operate big numbers

  
  - Import calculate to App component

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      break;
+    case 'X':
+      result = newNumberOne.times(newNumberTwo);
+      break;
+    case '÷':
+      result = numberTwo === 0 ? 'ERROR' : newNumberOne.div(newNumberTwo);
+      break;
+    case '%':
+      result = (newNumberOne.div(100)).times(newNumberTwo);
+      break;
```
Comment (trekab) [inline src/logic/operate.js]: `%` operator should turn the number into a percentage based number, `e.g. 95 % == .95`) which implies that it is a unary operator just like `+/-`.

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Well done with this milestone so far @MarilenaRoque! You just have one fix to work on to finish the milestone. Please fix your `%` operator's logic as indicated in the code comment below.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by MarilenaRoque** [comment]
Hello @trekab,
Thank you so much for the review!

### Required change Implemented :robot:

- [X] Percentage Operator transforms the last typed number into a percentage based number.

**Review by wathigo — APPROVED** [review]
### Approved :heavy_check_mark: 
Great job implementing the modifier operator functionality.

**You may proceed to the next milestone.**

**Happy Coding :computer: 
Simon Wathigo.**

**Comment by MarilenaRoque** [comment]
# Thank you so much for the review, @wathigo! 🚀

**Pull request merged**